_includes/macros/sub-pages.html

```diff
@@ -0,0 +1,92 @@
+
+   Sub-Pages
+
+   ==========================================================================
+
+   Description:
+
+   Builds sub-pages markup when given:
+
+   sub_pages      : An array containing sub-pags.
+
+   ========================================================================== #}
+
+{% macro render( sub_pages, show_in_office=false ) %}
+    {% set is_office_page = request.path.find('/offices/') > -1 %}
+    {% for page in sub_pages | sort(attribute='order') if is_office_page == false or
+                                                          page.show_in_office == "on" %}
+        {% if loop.first %}
+            <section class="office_sub_pages
+                          block
+                          block__sub
+                          block__padded-top
+                          block__border-top
+                          {{ "block__border-bottom" if is_office_page == false }}">
+                  <h1 class="h2">
+                      Our Work
+                  </h1>
+                  <div class="content-l
+                              content-l__main
+                              content-l__large-gutters
+                              expandables-group">
+        {% endif %}
+                      <div class="office_initiative
+                                  expandable
+                                  expandable__expanded
+                                  expandable__mobile-only
+                                  content-l_col
+                                  {# TODO: When we upgrade to Jinja2 2.8 we can use loop.length
+                                           to conditionally set this to one column or two. #}
+                                  content-l_col-1-2
+                                  ">
+                      <button class="expandable_header expandable_target u-show-on-mobile">
+                          {% if page.short_title or page.title %}
+                              <span class="expandable_header-left expandable_label">
+                                  {{ (page.short_title or page.title) | safe }}
+                              </span>
+                          {% endif %}
+                          <span class="expandable_header-right expandable_link">
+                              <span class="expandable_cue-open">
+                                  <span class="cf-icon cf-icon-plus-round"></span>
+                              </span>
+                              <span class="expandable_cue-close">
+                                  <span class="cf-icon cf-icon-minus-round"></span>
+                              </span>
+                          </span>
+                      </button>
+                      <div class="expandable_content">
+                          {% if page.short_title or page.title %}
+                              <h2 class="h4 u-hide-on-mobile">
+                                  {{ (page.short_title or page.title) | safe }}
```

Comment:
Possible move `(page.short_title or page.title)` to a variable since it's accessed on L:61, L:59, L:46, L:44.